v0/destinations/ga4/transform.js

```diff
@@ -41,6 +41,11 @@ const responseBuilder = (message, { Config }) => {
   }
 
   // trim and replace spaces with "_"
+  if (typeof event !== "string") {
+    throw new CustomError(
```

Comment:
We should use ErrorBuilder and set appropriate stat tags.

Reply:
There are lot of customErrors, we can make this change later.